ocw/lib/cleanup.py

```diff
@@ -32,5 +32,23 @@ def cleanup_run():
                 send_mail('{} on Cleanup in [{}]'.format(type(e).__name__, vault_namespace), traceback.format_exc())
 
 
+def list_clusters():
+    cfg = ConfigFile()
+    if cfg.has('clusters'):
+        for vault_namespace in cfg.getList(['cleanup', 'namespaces'], ['']):
+            try:
+                orphaned_clusters = EC2(vault_namespace).list_clusters()
```

Comment:
Why it is already `orphaned`? You just list clusters and if a test is already running, it isn't a left over? 
What piece I'm missing, plz give me some light!

Reply:
yeah you right :) statement is too bold :) the problem that I don't want to invest more time into this to come up with more smart logic where we will try to do some conclusions at least on timestamp :) 
anyway I am fine with any name which you like more than mine :)

Reply:
hm, what if I say, I don't like to get emails when ever someone run a test.
I really would say we should make some timestamp comparing here.